lib/task-data/schemas/dell-racadm-control.json

```diff
@@ -18,6 +18,18 @@
             "description": "Specify if reboot after racadm control tasks are done",
             "type": "boolean"
         },
+        "ip": {
```

Comment:
Schema in a file is usually defined in "definitions" and then used in "properties". If items are not specified in "properties" they won't take effect.
I saw you defined those items for IP, netmask, gateway, however I didn't see you use those definitions.